lib/Slider/Slider.js

```diff
@@ -384,12 +419,33 @@ module.exports = kind(
 		this.popupContentChanged();
 		this.tapAreaClassesChanged();
 		this.initSliderStyles();
+		this.jumpIncrementChanged();
 	},
 
 	/**
 	* @private
 	*/
-	destroy: function() {
+	initComponents: function () {
+		if (this.enableJumpIncrement) this.createJumpIncrementButton();
+		ProgressBar.prototype.initComponents.apply(this, arguments);
+	},
+
+	/**
+	* @private
+	*/
+	createJumpIncrementButton: function () {
+		this.createComponents(this.jumpWrapperComponents, {owner: this});
+		this.controlParent = this.$.sliderWrapper;
+		this.removeClass('moon-slider');
```

Comment:
Thank you @aarontam , during my commit, I made a confusing semantics. 
I read your suggestion and agree with it.
Let me cherry-pick your commit to this PR.